scripts/benchmarks/longbenchv2/bench.sh

```
@@ -0,0 +1,87 @@
+# SPDX-License-Identifier: Apache-2.0
+
+# LongBench-v2 evaluation script using sglang.test.run_eval
+
+head_node="localhost"
+head_port=8000
+model_name="nvidia/DeepSeek-R1-0528-NVFP4-v2"  # Default model name
+
+# Parse arguments from SLURM job
+n_prefill=$1
+n_decode=$2
+prefill_gpus=$3
```

Shellcheck (0.11.0)

[warning] prefill_gpus appears unused. Verify use (or export if used externally).

(SC2034)